ui/src/search.ts

```diff
@@ -6,6 +8,8 @@ import * as validation from "./validation";
 import type * as waitingRoom from "./waitingRoom";
 
 // STATE
+export const projectSearchInputStore: Writable<string> = writable("");
```

Comment:
Now we only support searching for projects but we may support other entities in the future, so I'd suggest renaming this to `inputStore`; shorter and its namespace already. Up to you :skateboard:

Reply:
86ff1adecb2a5cc6bb68d3813165ae9cd498b553